### PR TITLE
feat(all): add forgot password implementation

### PR DESCRIPTION
**Pull request by Jumer-Caragay**
- Add forgot password implementation
- Clean up lint errors
- Add disabled button ui look

### CODE REVIEW

**Comment by Jumer-Caragay** [comment]
- The email template looks different because I was testing sendgrid's design email templates

File: src/actions/index.js [hunk 1]
```diff
@@ -50,16 +50,41 @@ export const login = values => (dispatch) => {
...
-  //   })
-  //   .catch((error) => {
-  //     dispatch({ type: types.FORGOT_PASS_FAIL, error });
-  //   });
+  axios
+    .post(`${API_PATH}/forgot-password/request`, {
+      email: values.email,
+    })
+    .then(() => {
+      dispatch({ type: types.FORGOT_PASS_GUCCI });
```
Comment (selynna) [inline src/actions/index.js]: lmao forgot_pass_gucci

Comment (Jumer-Caragay) [reply src/actions/index.js]: ya @mianuddin did this, ngl pretty salty

**Comment by Jumer-Caragay** [comment]
> you may wanna handle the footer of the email though:
> image

I'm probably letting design handle the email templates

**Pull request merged**